src/main/java/uk/gov/hmcts/cmc/claimstore/ClaimStoreApplication.java

```diff
@@ -7,7 +7,7 @@
 
 @SpringBootApplication(exclude = DataSourceAutoConfiguration.class)
 @SuppressWarnings({"HideUtilityClassConstructor", "squid:S1118"}) // Spring needs a constructor, its not a utility class
-@EnableFeignClients
+@EnableFeignClients(basePackages = {"uk.gov.hmcts.cmc.claimstore", "uk.gov.hmcts.document"})
```

Comment:
Can we do `public static final String BASE_PACKAGE_NAME = "uk.gov.hmcts.cmc.claimstore"` below and use that constant here? Since we are forced to use the string would be nice to have only one way of getting the base package.

Reply:
Not possible :( Compiler says `Attribute value but me constant`